ledger/src/tests.rs

```diff
@@ -2743,23 +2729,139 @@ mod valid_solutions {
             // Set the latest block height.
             block_height = ledger.latest_height();
 
+            // Update the epoch solutions count.
+            if block_height % CurrentNetwork::NUM_BLOCKS_PER_EPOCH == 0 {
```

Comment:
```suggestion
            if block_height.is_multiple_of (CurrentNetwork::NUM_BLOCKS_PER_EPOCH) {
```

Pretty sure this is a clippy warning in newer Rust versions, so it would be good to address these now.

Reply:
Resolved with [6cca9ee](https://github.com/ProvableHQ/snarkVM/pull/3059/commits/6cca9eef8a0cd9fdacebfe34b69bba92f5681258)